features/portalAdmin/DARTS_Admin.feature

```diff
@@ -618,99 +618,3 @@ Feature: Admin portal
 
     Then I see "Audio file deleted" on the page
     And I do not see "{{seq}}-47" on the page
-
```

Comment:
Author note: this has been removed because it is a duplicate of the scenario above